web/src/frontend/src/nav-soknad/components/adresseAutocomplete/adresseAutcomplete.tsx

```diff
@@ -260,9 +260,26 @@ class AdresseAutocomplete extends React.Component<Props, StateProps> {
 		}
 	}
 
+	/*
+	 * Hack for å få museklikk på trefflisten i autocomplete til å fanges opp i Edge nettleser.
+	 */
+	handleItemClickInEdgeBrowser(event: any) {
+		const items: HTMLCollection = document.getElementsByClassName("item-highlighted");
+		if (items && items[0]) {
+			const INNER_TEXT = "innerText";
+			const selectedValue = items[0][INNER_TEXT];
+			this.state.adresser.map(adresse => {
```

Comment:
Adresser eller adresse kan ikke være null..? :)

Reply:
Hvis det ikke er noen adresser i this.state.adresser, så vil det ikke være noen treffliste under autocompletefeltet man kan klikke på. Tror ikke det har noe for seg å legge på en ekstra linje med nullpointer exception.